### ISSUE TITLE
JAWS loses sync with focused button - Blazor Auto template with prerendering disabled

### ISSUE THREAD

**Issue by code-dispenser** [FreedomScientific/VFO-standards-support#959]
**Summary**

JAWS loses its internal buffer/index synchronisation with the DOM. Despite visibly showing and announcing which button the user is on, after one or two presses it sends the synthesised click to a different focusable element on the page (when using specific Blazor settings).

In the video I did not move, I only navigated to the "Click Me And Increment A Counter" button and pressed it twice.
 
**Notes**
I eventually narrowed the issue down to the following specific condition:

You use the Blazor Web App template with Auto (Server and WebAssembly) render modes set per page, and the page in question has prerendering disabled. It does not occur with prerendering enabled, and it does not occur in any of the other Blazor templates (Global Server, Global WebAssembly, etc.).

I routinely test with the following assistive technologies on Windows 11: JAWS, Narrator and NVDA, each paired with Edge, Chrome and Firefox. I also test with Windows Voice Access. On macOS, iOS and Android I use VoiceOver on macOS paired with Safari, VoiceOver on iOS paired with Safari, and TalkBack paired with Chrome.

I have only experienced this issue with JAWS (it fails on all the browsers I test with) - no other screen reader is affected.

I have logged this issue against both the Blazor and JAWS repositories, given the specific nature of the interaction between the two.

**Expectations**

For JAWS to only send the click event to the element the user is currently interacting with.

**Reproduction**

Attached is a project using the default Blazor Web App template with Auto (Server and WebAssembly). GitHub would not let up upload the self-contained published app.

Visual Studio users please ensure the JAWSIssue project is set as the startup project.

**Versions**

I am currently using JAWS version 2026.2606.132 and Visual Studio 2026 18.8.2. The Blazor template used was the most current available as of today, 2026-08-02.

[JAWSIssue.zip](https://github.com/user-attachments/files/30633509/JAWSIssue.zip)

https://github.com/user-attachments/assets/51f31dd2-8645-4eb1-9f10-a4e624f8c854
